<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elevator Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1900</width>
         <height>1016</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,5">
        <item>
         <layout class="QFormLayout" name="mainMenuForm">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Building Settings</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="fireCheckBox">
            <property name="text">
             <string>Fire</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="fireSpinBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="powerOutageCheckBox">
            <property name="text">
             <string>Power Outage</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="powerOutageSpinBox"/>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>50</horstretch>
              <verstretch>25</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>2</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLineEdit" name="safeFloorLineEdit"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="safeFloorsLabel">
            <property name="text">
             <string>Safe Floors (comma seperated)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="elevatorLabel">
            <property name="text">
             <string>Elevators</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="elevatorSpinBox"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="passengerLabel">
            <property name="text">
             <string>Passengers</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="passengerSpinBox"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="floorLabel">
            <property name="text">
             <string>Floors</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="floorSpinBox"/>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QPushButton" name="updateSimulationButton">
            <property name="text">
             <string>Update Simulation</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QPushButton" name="startSimulationButton">
            <property name="text">
             <string>Start Simulation</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="elevatorSettingsLayout"/>
        </item>
        <item>
         <layout class="QGridLayout" name="passengerSettingsLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_window">
    <property name="title">
     <string>ElevatorSimulator</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuNew_window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
